viewport.cpp:1307:50: error - templates compilation error.

48 hours ago everything was compiling without errors.
Today compilation of templates after git pull always stops at:
```
scene/main/viewport.cpp: In member function 'void Viewport::_vp_input(const InputEvent&)':
scene/main/viewport.cpp:1282:50: error: 'class SceneTree' has no member named 'get_edited_scene_root'
  if (get_tree()->is_editor_hint() && get_tree()->get_edited_scene_root()->is_a_parent_of(this)) {
                                                  ^
scene/main/viewport.cpp: In member function 'void Viewport::_vp_unhandled_input(const InputEvent&)':
scene/main/viewport.cpp:1307:50: error: 'class SceneTree' has no member named 'get_edited_scene_root'
  if (get_tree()->is_editor_hint() && get_tree()->get_edited_scene_root()->is_a_parent_of(this)) {
```

Godot engine compiles without error in the same time.

I use xubuntu 14.04 LTS.

--- a/scene/main/viewport.cpp
+++ b/scene/main/viewport.cpp
@@ -1279,9 +1279,11 @@ void Viewport::_vp_input(const InputEvent& p_ev) {
 	if (disable_input)
 		return;
 
+#ifdef TOOLS_ENABLED
 	if (get_tree()->is_editor_hint() && get_tree()->get_edited_scene_root()->is_a_parent_of(this)) {
 		return;
 	}
+#endif
 
 	if (parent_control && !parent_control->is_visible())
 		return;
@@ -1304,9 +1306,11 @@ void Viewport::_vp_unhandled_input(const InputEvent& p_ev) {
 	if (disable_input)
 		return;
 
+#ifdef TOOLS_ENABLED
 	if (get_tree()->is_editor_hint() && get_tree()->get_edited_scene_root()->is_a_parent_of(this)) {
 		return;
 	}
+#endif
 
 	if (parent_control && !parent_control->is_visible())
 		return;

